### ISSUE TITLE
[DECOHack] Add Boom DeHackEd Sound Fields, Maybe

### ISSUE THREAD

**Issue by MTrop** [MTrop/DoomTools#54]
So apparently, a long time ago, Boom introduced a gaggle of undocumented sound extensions like Initial Pitch and Volume and other things into its DeHackEd. It has made its way into other ports, but it is not widely implemented.

From Eternity:
```
static constexpr const char *deh_sfxinfo[] =
{
  "Offset",     // pointer to a name string, changed in text
  "Zero/One",   // .singularity (int, one at a time flag)
  "Value",      // .priority
  "Zero 1",     // .link (sfxinfo_t*) referenced sound if linked
  "Zero 2",     // .pitch
  "Zero 3",     // .volume
  "Zero 4",     // .data (SAMPLE*) sound data
  "Neg. One 1", // .usefulness
  "Neg. One 2"  // .lumpnum
};
```

Until we know how widespread its use is, this will not be implemented, but I'm just making this issue for awareness, since it seems to be at the very least parse-able from Boom-compatible ports and onward.

**Comment by MTrop** [comment]
This may be potentially implemented as a DECOHack feature via enhancement #55. 

This issue should be closed upon its completion.

**Comment by MTrop** [comment]
Added with the addition of custom properties, more or less. Commit 945da0932c9b5d45f130e8df3a125e3ed23445c8.
